### PR TITLE
Updating dotnet-watch to netstandard

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

**Comment by muratg** [comment]
Looks good to me. I'll let @victorhurdugaci give the squirrel.

**Comment by moozzyk** [comment]
This can't be merged until https://github.com/aspnet/CoreCLR/issues/79 is fixed. /cc @pranavkm

**Comment by victorhurdugaci** [comment]
:shipit:

**Comment by victorhurdugaci** [comment]
We also need a newer CLI. It seems that the one that we have doesn't support "platform" in `project.json`. (see the travis run)

**Comment by muratg** [comment]
We have a new CLI now which supports this. @anurse, @NTaylorMullen and @pranavkm were prototyping this earlier. Not sure if there are still blockers.

**Comment by moozzyk** [comment]
I am testing publish-iis and it seems to work so I assume this should work too.

**Comment by moozzyk** [comment]
:up: :date:

**Comment by victorhurdugaci** [comment]
Don't you have to merge this in release?

File: test/dotnet-watch.FunctionalTests/Scenario/ProjectToolScenario.cs [hunk 1]
```diff
@@ -15,12 +15,14 @@ public class ProjectToolScenario: IDisposable
     {
         private const string NugetConfigFileName = "NuGet.config";
 
+        private static object _restoreLock = new object();
```
Comment (victorhurdugaci) [inline test/dotnet-watch.FunctionalTests/Scenario/ProjectToolScenario.cs]: readonly

**Comment by muratg** [comment]
Looks good!  Make the lock readonly, and merge to release branch (:shipit:)

**Comment by victorhurdugaci** [comment]
It failed on OSX (Travis) :cry:

**Comment by moozzyk** [comment]
I saw the Travis failure - I think I will look into separately.

**Pull request merged**